pkg/didcomm/protocol/introduce/service.go

```diff
@@ -21,6 +23,9 @@ const (
 	Introduce = "introduce"
 	// IntroduceSpec defines the introduce spec
 	IntroduceSpec = metadata.AriesCommunityDID + ";spec/introduce/1.0/"
+	// SkipProposalMsgType defines the skip proposal (the introducer has a public invitation)
+	// skip proposal is not part of protocol specification (internal usage)
+	SkipProposalMsgType = "skip/proposal"
```

Comment:
when we spoke about this you mentioned it was going to be some sort of internal implementation detail. Therefore this should definitely not be exported.

Reply:
the problem is that there is no way to know that it is skip proposal

Reply:
I am replacing it to real one https://github.com/hyperledger/aries-framework-go/pull/516/files#diff-43cf3af5d3c6b9acebc5c635ba4d0f7dR174

Reply:
when the client perform this action we need to know somehow that is Skip proposal (and I found the way to create this fake type and then replace it to real one)

Reply:
any suggestion welcome :) but I don't think it is a good idea to extend DIDComm message or proposal model just bc we need to know that this is skip proposal. Or we can do some injection to the DB which is the worst solution ever